src/common/zstd_compression.cpp

```diff
@@ -5,6 +5,7 @@
 #include <algorithm>
 #include <zstd.h>
 
+#include "common/assert.h"
```

Comment:
`common/assert.h` has the definitions for `ASSERT`, `UNREACHABLE` and `UNIMPLEMENTED`, and I don't see them used here.
It works because `assert.h` includes `log.h`. Try:
```suggestion
#include "common/logging/log.h"
```

Reply:
Thank you for the suggestion. It worked like a charm.

On an unrelated note, I have also added a second change that fixed another failure that cropped up on a fresh build.

Reply:
It seems the suggested change got lost at some point when you added the second change

Reply:
No idea how that happened. Fixed.